velox/connectors/hive/storage_adapters/hdfs/tests/HdfsFileSystemTest.cpp

```diff
@@ -152,7 +145,6 @@ void verifyFailures(ReadFile* readFile) {
           .str();
   checkReadErrorMessages(readFile, offsetErrorMessage, kOneMB);
   HdfsFileSystemTest::miniCluster->stop();
-  checkReadErrorMessages(readFile, readFailErrorMessage, 1);
```

Comment:
Why removing this check?

Reply:
If we want to verify open file errors, we should ensure that the target file has not yet opened the file handle, adjusted UT to satisfy this.